docs/sources/alerting/fundamentals/notifications/_index.md

```diff
@@ -1,6 +1,8 @@
 ---
-canonical: https://grafana.com/docs/grafana/latest/alerting/fundamentals/notification-policies/
-description: Learn about how notification policies work
+aliases:
+  - ./notification-policies/
```

Comment:
Please add the comment with the absolute URL this redirects to make this easier for the future.

Reply:
fixed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344